internal/artifacts/testdata/mtahtml5/mta.yaml

```diff
@@ -12,6 +12,8 @@ modules:
    requires:
     - name: uaa_mtahtml5
     - name: dest_mtahtml5
+   build-parameters:
```

Comment:
Why did you add this?

Reply:
in order to pass successfully setManifest if called with unsupported platform. To have an option to succeed with wrong paths till mtad generation. Maybe trick is wrong and we have to remove unsupported (by platform) modules from mtad...

Reply:
Sorry, I don't understand... will a test fail if you remove it? If so, which test and why? And why is it different from before these fixes?

Reply:
After discussion - the test sends neo instead of cf, so the metadata generation ignores the empty module paths, and the mtad generation fails on them